tests/ui/coroutine/const_gen_fn.stderr

```diff
@@ -0,0 +1,20 @@
+error: functions cannot be both `const` and `gen`
+  --> $DIR/const_gen_fn.rs:6:1
+   |
+LL | const gen fn a() {}
+   | ^^^^^-^^^----------
+   | |     |
+   | |     `gen` because of this
+   | `const` because of this
```

Comment:
This seems so obvious make it feels a bit redundant. 😄

Reply:
Yeah... preexisting though